fpo-web/src/app/survey/primary.component.html

```diff
@@ -9,10 +9,15 @@
         [initialMode]="initialMode"
       ></app-survey-view>
     </div>
-    <div class="col-md-12" *ngIf="submitted">
+    <div class="col-md-12 mt-4" *ngIf="submitted">
       <p><b>Thank-you for choosing how you wish to attend your traffic hearing.</b></p>
-    
-      <p>A copy of your completed form is attached for your records.</p>
+       
+      <div id="emailSent" *ngIf="emailStatus; else emailFailed">
```

Comment:
With this logic, it will always show failed.. until the ajax comes back from submit-form. I'd recommend hiding this template until submit-form returns.

Reply:
![image](https://user-images.githubusercontent.com/3484109/90298275-7fb70800-de46-11ea-94f0-2151608240e0.png)